### DISCUSSION TITLE
part-ii/2019-77/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#171] [Question comments]
Part II 2019 Graph Theory: Paper 2, Section II, 17G 

https://questions.tripos.org/part-ii/2019-77/

**Comment by stevezshen** [comment]
<details>
<summary>Spoiler warning</summary>

c) induction on t. Find successively t monochromatic triangles of the same color, or there are a blue and a yellow vertex disjoint triangles, and delete those five vertices and apply induction hypothesis. 

</details>
